codebase-stewardship/index.html

```diff
@@ -17,58 +17,58 @@
   </p>
 
   </div>
   <img alt="illustration, interconnecting componements" src="https://publiccodenet.github.io/illustrations/illustrations/interconnecting-componements.svg">
 </div>
 
 <div class="mp-section">
-  <div><h2>Enabling the community to grow, organize and build amazing things together</h2>
+  <div><h2>Enabling communities to grow, organize and build amazing things together</h2>
 
   <p>
-    With <strong>community development</strong> we help the community organize and grow, process feedback and contributions, host events, and build and execute effective governance.
+    We co-develop communities to help them process feedback and contributions, host events, and build and execute effective governance.
   </p>
   </div>
   <img alt="illustration, community" src="https://publiccodenet.github.io/illustrations/illustrations/community-2.svg">
 </div>
 
 <div class="mp-section">
-  <div><h2>Adding the stories and details a codebase needs to be noticed and re-used</h2>
+  <div><h2>Building the story a codebase needs to be noticed and re-used</h2>
 
   <p>
-    Through <strong>product management</strong> we grow adoption and help mature the codebase to the point where it can shine on its own. So potential re-users and future contributors notice and adopt the codebase, we help turn it in to a product with strong branding and communications. We also support on development and design.
+    We manage and mature products to help them shine on their own, build branding and plan communications to ensure codebases get adopted by re-users and contributors, and support both development and design.
   </p>
   </div>
   <img alt="illustration, social software" src="https://publiccodenet.github.io/illustrations/illustrations/social-software.svg">
 </div>
 
 <div class="mp-section">
-  <div><h2>Providing users, vendors and developers with the tools, skills and answers they need</h2>
+  <div><h2>Providing tools, skills and answers for users, vendors and developers</h2>
 
   <p>
-    To drive <strong>developer success</strong> we help with (end)user support, packaging, and distributing official versions as well as developing training materials and certifications. We also help vendors such as cooperatives, NGOs and for profit companies to sell, implement and support the product.
+    We help developers with end-user support, packaging and distribution, training material creation and certifications, while helping cooperatives, NGOs and for-profit companies sell, implement and support products.
   </p>
   </div>
   <img alt="illustration, ecosystem" src="https://publiccodenet.github.io/illustrations/illustrations/eco-system.svg">
 </div>
 
 <div class="mp-section">
-  <div><h2>We take care of the product and codebase's operations and legal affairs</h2>
+  <div><h2>Taking care of operations and legal affairs</h2>
```

Comment:
I had missed that we are *taking care of* legal affairs (as it is missing on our [activities](https://about.publiccode.net/activities/codebase-stewardship/activities.html). We do support codebases in this, but 'taking care of' sounds more than that.

Reply:
Indeed, I can see that something like "assist" is better for legal.

Reply:
I had a discussion with @bvhme about this at some point; he argued that the license advice he was then giving provided enough justification for this strong assertion.

Reply:
Might I perhaps add that I think it might be worth it for us to hire a lawyer when a community faces a legal challenge as a collective more than leaving this onto individual members. With that we might be able to take on some of the liability of the communities to take that argument away as a reason for not using those tools.

This is also commonly done by other open source foundations, what would make us different?

I'm curious as to what the @publiccodenet/codebase-stewards think about this in general.

Reply:
In the interest of iterative working, I think this should be left as a softer 'supporting' for now.